### ISSUE TITLE
[BUG] - Problems while uploading files

### ISSUE THREAD

**Issue by xavialex** [code-kern-ai/refinery#65]
Hi everyone. I'm afraid I'm experiencing the same error as [this one](https://github.com/code-kern-ai/refinery/issues/64#issue-1325827457): once refinery is launched, in the GUI, while creating a new project, the Upload file functionality is disabled (clicking doesn't allow to select any files). I'm not sure if this is the same problem, since I'm working on a Mac and the IP adress seems to be correctly gathered. I'm also using the self-hosted version.

Any thoughts on this?. Thanks in advance

**Comment by jhoetter** [comment]
Hi @xavialex! 
I will assign this to @SirDegraf too, as he is more familiar with upload issues. To better understand, do you mean that there is no file browser appearing as you click on the file selector (i.e. "Click to select a file" has no interaction)?
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/62521561/182598914-c096b1ba-ac3c-465e-84fa-ce1e2d649495.png">

Which browser are you using? (Safari, Chrome, ..)

**Comment by xavialex** [comment]
Exactly. Drag and drop doesn't work either. It's strange because I managed to upload CSV files yesterday while making my first tests with the platform. It's also not working for adding more data into an already created project.

**Comment by jhoetter** [comment]
That's really a bit strange. Could you please tell me if the error is still there after a restart in the refinery repository? I.e. running `docker-compose -f refinery/docker-compose.yml restart` inside the repository.

If it already worked, I can only assume that something with Docker is not working as it should. And usually, a restart helps here. However that shouldn't happen after only 24 hours, so we'll have to look into this further. But for now, maybe this helps?

**Comment by xavialex** [comment]
Hi jhoetter, thanks for your answer. It didn't work (same behaviour as previously) but restarting the machine did. For more information, it is a MacBook M1 Pro. I've noticed hower that there are two containers out of the two associated with refinery that are not running, refinery_kratos-migrate_1 and refinery_postgres-migrate_1. Closing this for now but let me know if I can provide more info for help.
